verification/abi.go

```diff
@@ -437,6 +471,32 @@ func (c *dpeABI[_, Digest]) CertifyKey(handle *ContextHandle, label []byte, form
 	return key, nil
 }
 
+func (c *dpeABI[_, Digest]) Sign(handle *ContextHandle, label []byte, flags SignFlags, toBeSigned []byte) (*DPESignedHash, error) {
+	cmd := SignReq[Digest]{
+		ContextHandle: *handle,
+		Label:         Digest(label),
+		Flags:         flags,
+		ToBeSigned:    Digest(toBeSigned),
+	}
+
+	if len(label) != len(cmd.Label) {
+		return nil, fmt.Errorf("Invalid digest length")
+	}
```

Comment:
nit: Move this to the beginning, before calling `Digest(label)`

Reply:
Added the check